content/faq/editing/advanced-map-editing/index.md

```diff
@@ -0,0 +1,79 @@
+
+As all map data used in Organic Maps comes from [OpenStreetMap.org (OSM)](https://www.openstreetmap.org), you can directly update the map there. Your modifications will then be included into Organic Maps with the next map update.
+
+## OpenStreetMap Editors
+
+For editing OSM, there are several options. If you have a desktop PC at hand, it is recommended that you use the ID Editor. The editor is easy to use for beginners, and a bigger screen, mouse, and keyboard make editing easier. The editor runs in your browser, so no installation is required.
+
+If you don't have a desktop PC at hand, you can use the mobile editors Go Map for ios or Vespucci for Android. Go Map is easy to use for beginners, while Vespucci targets more advanced users.
+
+Please pick one of the three editors to proceede.
+
+#### ID Editor
+For editing OpenStreetMap with ID follow these steps:
+
+1. Create a new account or log in at [OpenStreetMap.org](https://www.openstreetmap.org)
+2. Browse to the location you want to edit on OpenStreetMap.org and click *Edit*
+3. *Start the Walkthrough* and follow the short tutorial that explains the ID Editor
+4. Edit the map
+5. Upload your changes
+
+That's it, you are now part of the OSM community. Wait for the next Organic Maps update to see the changes in your favorite map app.
+
+#### Go Map
+For editing OpenStreetMap with Go Map follow these steps:
+
+1. Download the app from the [App Store](https://apps.apple.com/de/app/go-map/id592990211)
+2. Follow the detailed [LearnOSM tutorial](https://learnosm.org/en/mobile-mapping/gomap/)
+
+#### Vespucci
+For editing OpenStreetMap with Vespucci follow these steps:
+
+1. Download the app from the [Play Store](https://play.google.com/store/apps/details?id=de.blau.android) or from [F-Droid](https://f-droid.org/en/packages/de.blau.android/).
+2. Follow the detailed [LearnOSM tutorial](https://learnosm.org/en/mobile-mapping/vespucci/)
+
+## What happens with my edits?
+Once you press *Upload* your changes are instantly added to the public OSM database. So be considerate when editing. Your e-mail is not published, but other people will be able to see your OSM user name. 
+
+As OSM offers the possibility to discuss changes, you might get questions about your edits from other OSM contributors. You will be notified about this via the e-mail address you used for registering your OSM account. As OSM is a community project that builds on collaboration you should always answer such questions.
+
+## Community and Wiki
+
+OpenStreetMap is a community. If you need help you can ask in the [OSM Forum](https://community.openstreetmap.org/) or take a look at the [OSM Wiki](https://wiki.openstreetmap.org/) documentation.
```

Comment:
Any live discussion channels? List of telegram/other messenger groups? Remember, our audience is not a geeky one.

Reply:
The help and support page in the OSM Forum is the main place for questions. But there are also other channels (see: https://wiki.openstreetmap.org/wiki/Contact_channels). I haven't tried out any of them though.